Rubberduck.CodeAnalysis/Rubberduck.CodeAnalysis.csproj

```diff
@@ -8,6 +8,9 @@
     <RootNamespace>Rubberduck.CodeAnalysis</RootNamespace>
     <ProjectGuid>{DEF2FB9D-6E62-49D6-8E26-9983AC025768}</ProjectGuid>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <DocumentationFile>C:\Dev\GitHub\Rubberduck\Rubberduck.CodeAnalysis\Rubberduck.CodeAnalysis.xml</DocumentationFile>
```

Comment:
Should that really be an absolute path?

Reply:
This only gets generated for Debug builds. This should not be dependent on `Configuration` *or* `Platform`.
Also this restriction is what's breaking the PostBuild event's `xcopy` on CI, because the file is not generated for the Release build that AppVeyor performs